extensions/cl_ext_cxx_for_opencl.asciidoc

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018-2020 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+:data-uri:
+:icons: font
+include::../config/attribs.txt[]
+:source-highlighter: coderay
+
+= cl_ext_cxx_for_opencl
+:R: pass:q,r[^(R)^]
+Khronos{R} OpenCL Working Group
+
+== Name Strings
+
+`cl_ext_cxx_for_opencl`
+
+== Contact
+
+Please see the *Issues* list in the Khronos *OpenCL-Docs* repository: +
+https://github.com/KhronosGroup/OpenCL-Docs
+
+== Contributors
+
+Kevin Petit, Arm Ltd. +
+Sven Van Haastregt, Arm Ltd. +
+Anastasia Stulova, Arm Ltd. +
+Marco Antognini, Arm Ltd. +
+Neil Hickey, Arm Ltd. +
+
+== Notice
+
+include::../copyrights.txt[]
+
+== Status
+
+Draft
```

Comment:
When will we change the status?

Reply:
The status makes less sense for an EXT (no formal approval or ratification), I'll remove the section entirely.